src/EventSubscriber/RateLimitSubscriber.php

```diff
@@ -0,0 +1,111 @@
+<?php
+declare( strict_types=1 );
+
+namespace App\EventSubscriber;
+
+use DateInterval;
+use Krinkle\Intuition\Intuition;
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
```

Comment:
Unused use statement.

Reply:
I think it would've been caught by the `MediaWiki.Classes.UnusedUseStatement.UnusedUse` sniff, which we're already using. It didn't only because CI didn't get that far, I think.